### DISCUSSION TITLE
Lincença

### DISCUSSION THREAD

**Discussion by Daniel1412536** [aclap-dev/video-downloadhelper#988] [License & Purchase requests]
### What's your issue?

I bought the license for the wrong browser, I would like to know if you could send me the license for Chrome, as I don't use Fire Fox

### The 10 first digits (only 10) of your license number or email address

100e9d726d

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry we can't swap licenses, Chrome and Firefox premium are separate products with different prices and features.
What I can do is refund your Firefox premium so that you can purchase Chrome premium if you want to.
Do you want me to do this ?
